### DISCUSSION TITLE
Create samples for sentimental analysis

### DISCUSSION THREAD

**Discussion by sergey-tihon** [sergey-tihon/Stanford.NLP.NET#111] [Ideas]
[Deeply Moving: Deep Learning for Sentiment Analysis](http://nlp.stanford.edu/sentiment/index.html)

**Comment by JimSEOW** [comment]
good idea

**Comment by JimSEOW** [comment]
The upcoming Stanforld-NLP CoreNLP version e.g. 3.9: is now tracking Sentiment Analysis support for [other languages](https://github.com/stanfordnlp/CoreNLP/blob/master/src/edu/stanford/nlp/simple/GermanDocument.java)
